docs/product/release.rst

```diff
@@ -70,7 +70,8 @@ driving the release, and that person will be the **release manager**.
 
 Some functions of the **release manager** are:
 
-* Defining the milestone description and due date
+* Defining the milestone title (SFTPPlus-MAJOR.MINOR.0), description and
```

Comment:
Instead of `SFTPPlus-MAJOR.MINOR.0` maybe use `PRODUCT_NAME-MAJOR.MINOR.0`

right now we have it only for SFTPPlus but in the future we might have other products, ex AtriaMFT and I want this information to be valid for all products